docs/scheduling.md

```
@@ -0,0 +1,240 @@
+    "conversationStrategy": "persistent",
+    "message": "heartbeat ping",
+    "timeZone": "UTC",
+    "enabled": true
+  }'
+```
+
+### Cron Expression Reference
+
+EDDI uses **standard 5-field cron expressions**:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)